addons/project/views/project_views.xml

```diff
@@ -1436,15 +1433,10 @@
                                             <field name="milestone_id" options="{'no_open': True}" />
                                         </span>
                                         <br />
+                                        <!-- to check  -->
                                         <t t-if="record.partner_id.value">
-                                            <span t-if="!record.partner_is_company.raw_value">
-                                                <field name="commercial_partner_id"/>
-                                            </span>
-                                            <span t-else="">
-                                                <field name="partner_id"/>
-                                            </span>
+                                            <field name="partner_id"/>
```

Comment:
Does this still work as before?

Reply:
no check first point from the commit body